interfaces/builtin/polkit.go

```diff
@@ -148,6 +149,28 @@ func (iface *polkitInterface) BeforePreparePlug(plug *snap.PlugInfo) error {
 	return err
 }
 
+func isPathMountedWritable(mntProfile *osutil.MountProfile, fsPath string) bool {
```

Comment:
Maybe this could go to `osutil`, it is quite generic, and maybe it should load itself the mounts.

Reply:
I see this as a great idea for a followup so we can get the fix landed - I'll work on a version moved to osutil, and then it can load the mounts itself (I couldn't in this version as it was loaded by init() making it hard to mock the path)

Reply:
Sounds good, approving now